client/blocks/plugins-update-manager/schedule-form.helper.ts

```diff
@@ -34,10 +36,11 @@ export const prepareTimestamp = (
  */
 export const validateName = ( name: string ) => {
 	let error = '';
+
 	if ( ! name ) {
-		error = 'Please provide a name to this plugin update schedule.';
+		error = translate( 'Please provide a name to this plugin update schedule.' );
 	} else if ( name.length > 120 ) {
-		error = 'Please provide a shorter name.';
+		error = translate( 'Please provide a shorter name.' );
```

Comment:
We shouldn't need these?

Reply:
Agree

Reply:
I've removed the `validateName` function since we removed the Name control from the form.